README.md

```diff
@@ -32,6 +32,17 @@ kubectl port-forward my-release-marklogic-0 8000:8000 8001:8001
 ```
 Please refer [Documentation](http://cms-ml-docs-stage.marklogic.com/11.0/guide/kubernetes-guide/en/creating-a-marklogic-cluster/accessing-marklogic-server-in-a-kubernetes-cluster.html) for more options on accessing MarkLogic server in a Kubernetes cluster.
 
+By default, MarkLogic admin username will be set to `admin` and the password will be set to a 10-character long random alphanumeric value. Use the following steps to extract the admin password:
+
+1. Run the below command to fetch the secret name for MarkLogic deployment
+``` 
+kubectl get secrets 
+```
+2. Use the secret name from step 1 to get MarkLogic admin password 
+``` 
+kubectl get secret SECRET_NAME -o jsonpath='{.data.password}' | base64 --decode 
```

Comment:
How do they know what secret to use?

Reply:
Added instruction on the format of secret name to use in the command.